### PR TITLE
[PWX-33941, PWX-33951]Added Pool expand with maintenance mode tests.

### PR DESCRIPTION
**Pull request by vinayakshnd**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:
Will update jenkins job runs here once they finish successfully.

### CODE REVIEW

File: tests/basic/pool_function_test.go [hunk 1]
```diff
@@ -535,3 +535,143 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
...
+		Steps:
+			1. Move a node to maintenance mode.
+			2. Initiate pool expand with resize-disk operation.
+			3. Exit out of maintenance mode (PX only performs pool expand in normal mode, not in maintenance mode)
+			4. Verify pool expand operation goes to completion.
+	*/
+
+	BeforeEach(func() {
+		contexts = scheduleApps()
+		StartTorpedoTest("PoolExpandResizeDiskInMaintenanceMode",
```
Comment (lsrinivas-pure) [inline tests/basic/pool_function_test.go:553]: make StartTorpedoTest as the first step

File: tests/basic/pool_function_test.go [hunk 2]
```diff
@@ -535,3 +535,143 @@ var _ = Describe("{PoolExpandWhileResizeDiskInProgress}", func() {
...
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+
+	It("Start pool expand with resize-disk on node which is already in maintenance mode ", func() {
```
Comment (lsrinivas-pure) [inline tests/basic/pool_function_test.go:572]: can you push these steps also to aetos using log.infoD

Comment (vinayakshnd) [reply tests/basic/pool_function_test.go]: Done

File: tests/basic/pool_function_test.go (hunk 1 above)
Comment (vinayakshnd) [reply tests/basic/pool_function_test.go:552]: Done

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**